src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -613,8 +616,8 @@ def _validate(target_iqn, target_controls, portals, disks, groups, settings):
         # When using an older `ceph-iscsi` version these validations will
         # NOT be executed beforehand
         IscsiTarget._validate_target_controls_limits(settings, target_controls)
-        portal_names = [p['host'] for p in portals]
-        validate_rest_api(portal_names)
+        # portal_names = [p['host'] for p in portals]
+        # validate_rest_api(portal_names)
```

Comment:
What's this for? Useless IMO. The check has been done already some lines above by calling `get_active_portal` which effectively returns the active gateway with the iSCSI REST API.